core/controllers/reader.py

```diff
@@ -329,12 +329,19 @@ def post(self, unused_exploration_id):
         """
         # A domain object representing the old state.
         old_state = exp_domain.State.from_dict(self.payload.get('old_state'))
+        # The id of the exploration to which the state belongs.
+        exp_id = self.payload.get('exp_id')
+        # The version of the exploration when the classifier model was created.
+        exp_version_when_created = self.payload.get('exp_version')
```

Comment:
@anmolshkl -- aren't the current version of the exploration, and the exp_version_when_created, different things? How do we derive the latter from the former -- or do we just assume that the latest exp_version_when_created applies?